style.css

```diff
@@ -88,4 +91,212 @@
     margin-left: 0;
     margin-top: 32px;
   }
+
+  /* sections */
+  .projects {
+    display: flex;
+    flex-direction: column;
+  }
```

Comment:
[REQUIRED] Although I like the layout you implemented for your `.projects` section using `flexbox`. I have to ask you to use `Grid` for this section (the main container -  `.projects`) as it is a must by this milestone. please refer to the below project requirement:

![Screenshot from 2022-02-17 23-38-50](https://user-images.githubusercontent.com/67608876/154548564-1c23dc71-164b-4fa7-b0e4-70f6384bd1a5.png)

We just want to make sure you understand both concepts (`Grid` and `flexbox`) well since they are very powerful tools and once you master them setting the `layout` will be very easy.

Reply:
ALl good now 👍🏽